www/pages/project-advanced.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: page
+permalink: /project-advanced
+title: Transfer or delete a project
+---
+
+To access the advanced project settings:
+
+- Select **SETTINGS** from above the project files
+- At the bottom of the page, you have three options:
+  - Public/private setting: to make your project public or private
```

Comment:
It would be good to add a link to `/permissions` page for explanation of the difference.